MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include *.md
+include *.txt
+recursive-include cf_units *.py
```

Comment:
I don't think you need this because the `cf_units` and `cf_units.tests` packages are already declared in `setup.py`. Probably doesn't do any harm though!